src/superClient.js

```diff
@@ -27,8 +30,19 @@ module.exports = async function superClient(
   } else if (basicAuth) {
     suObj = suObj.auth(basicAuth.username || '', basicAuth.password || '');
   }
-  return suObj
-    .set(req.header || '')
-    .send(data)
-    .sortQuery();
+
+  suObj.set(req.header || '');
+
+  if (proxy) {
+    suObj.proxy(proxy);
+  }
+
+  // Only set data if data has some value, so that content-length is zero
+  if (data && data !== 'null' && Object.keys(data).length > 0) {
+    suObj.send(data);
+  }
```

Comment:
for content-length Zero, to fix the issue when some GCK load balancer was not accepting requests if the content length was not correct